nanodbc/nanodbc.cpp

```diff
@@ -3743,7 +3743,9 @@ connection::connection(const string& connection_string, long timeout)
 {
 }
 
-connection::~connection() noexcept {}
+connection::~connection() noexcept
+{
+}
```

Comment:
Does this and the rest reformatting come from our `.clang-format`?

Reply:
Used the style script, but from within the container (guess different clang-format version). When I ran the script from the host os, then it changed back these lines.